app/code/Magento/Quote/Model/Quote/Address.php

```diff
@@ -1452,6 +1453,9 @@ public function setCountryId($countryId)
     public function getStreet()
     {
         $street = $this->getData(self::KEY_STREET);
+        if ($street === null) {
+            return [''];
```

Comment:
Hello @karyna-tsymbal-atwix 
Maybe, it would be better to use `[]` here.
What do you think about it?

```suggestion
            return [];
```

Reply:
@andrewbess  I think this public function can be used very actively in custom modules.
I think it's better to preserve the old behaviour.

Reply:
I would suggest to use `$street = $this->getData(self::KEY_STREET) ?? [''];` and have single `return` statement

Reply:
Hello @xmav 
Great idea
Fixed? please check